contribs/gmf/src/directives/partials/authentication.html

```diff
@@ -0,0 +1,165 @@
+<div ng-if="gmfUser.username">
+  <div class="form-group">
+    <span>{{ authCtrl.translate('Logged in as') }}</span>
+    <strong>{{ gmfUser.username }}</strong>.
```

Comment:
Unless I misunderstand what a one time binding does, shouldn't this not be a one time binding ?  What if we log in a second time using an other username ?  Would a one time binding show the second username ?